alpha-solver/src/test/java/at/ac/tuwien/kr/alpha/regressiontests/SolverTests.java

```diff
@@ -805,16 +803,17 @@ public void smallCardinalityAggregate(RegressionTestConfig cfg) {
 		);
 	}
 
-	@RegressionTest
-	public void dummyGrounder(RegressionTestConfig cfg) {
-		AtomStore atomStore = new AtomStoreImpl();
-		assertEquals(GrounderMockWithBasicProgram.EXPECTED, buildSolverForRegressionTest(atomStore, new GrounderMockWithBasicProgram(atomStore), cfg).collectSet());
-	}
+	// TODO @AntoniusW what are these? Can we get rid of them? If not, where do I move them?
```

Comment:
@AntoniusW see `TODO`: The commented-out test cases don't seem to be following the pattern of an end-to-end system test of all other tests in this class. Should we move these to some other location and remove the parameterization?